index.js

```diff
@@ -1,18 +1,9 @@
-let elements = document.getElementsByTagName('*');
-
-for (let i = 0; i < elements.length; i++) {
-    let element = elements[i]
-
-    for (let j = 0; j < element.childNodes.length; j++) {
-        let node = element.childNodes[j];
-
-        if (node.nodeType === 3) {
-            let text = node.nodeValue;
-            let replaced = text.replace('69', 'Nice 😎');
-
-            if (replaced !== text) {
-                node.nodeValue = replaced;
-            }
+const elements = document.getElementsByTagName('*');
+const elementsArray = Array.from(elements);
+elementsArray.forEach(element => {
```

Comment:
I might do `[...elements].forEach()` but that's literally just stylistic.

Reply:
this is hot